pandas/tests/frame/test_analytics.py

```diff
@@ -2067,6 +2067,58 @@ def test_dot(self):
         with tm.assert_raises_regex(ValueError, 'aligned'):
             df.dot(df2)
 
+    def test_matmul(self):
+        a = DataFrame(np.random.randn(3, 4), index=['a', 'b', 'c'],
```

Comment:
instead of writing a new test, just parameterize test_dot on dot/__matmul__ as this is pretty much the same

Reply:
you will need to skip the matmul tests if not PY35